From 73a76a3be4204d279a098d0b14ffb1817139e4a8 Mon Sep 17 00:00:00 2001
From: Roy Pledge <Roy.Pledge@freescale.com>
Date: Thu, 14 Apr 2016 10:21:02 +0800
Subject: [PATCH 057/151] fsl_qbman: Enable DPAA1 QBMan for ARM64 platforms

commit ddb0ae0f9d638efc70402d4fb570bca3a3178091
[context adjustment]

Enable the QBMan device for ARM64 platofrms.  This is needed for
ARM based SoCs that have DPAA1 such as the LS1043A

Signed-off-by: Roy Pledge <Roy.Pledge@freescale.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/Kconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/Kconfig b/arch/arm64/Kconfig
index bf4c50b..c7314c2 100644
--- a/arch/arm64/Kconfig
+++ b/arch/arm64/Kconfig
@@ -263,6 +263,11 @@ endmenu
 
 menu "Bus support"
 
+config HAS_FSL_QBMAN
+        bool "Datapath Acceleration Queue and Buffer management"
+        help
+          Datapath Acceleration Queue and Buffer management
+
 config PCI
 	bool "PCI support"
 	help
-- 
2.1.0.27.g96db324

